src/index.js

```diff
@@ -13,11 +18,13 @@ ReactDOM.render(
   <React.StrictMode>
     <Router>
       <AuthProvider>
-        <VideoProvider>
-          <SidebarProvider>
-            <App />
-          </SidebarProvider>
-        </VideoProvider>
+        <PlaylistProvider>
```

Comment:
could separate out all of the providers in different component

Reply:
Thanks for the suggestion. Will do that.